### DISCUSSION TITLE
My main account has been flagged as spam

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23865] [New to GitHub]
<p>Yeah, hello, my main GitHub account (ThatNormalTet) has been flagged as spam for unknown reasons.</p>
<p>I say “unknown reasons” because i literally havent used it since i created it, just this few days to help a friend with a project.</p>
<p>im aware this isnt the contact support, just a forum, but i dont know how to contact the people who can get rid of this flag, so please, help.</p>
<p>Thanks</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The form for account issues is here: <a href="https://support.github.com/contact/account" class="inline-onebox">Accounts - GitHub Support</a></p>
